app/subscriber/src/features/filter-media/styled/FilterMedia.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+
+export const FilterMedia = styled.div`
+  .tool-bar {
+    .reset {
+      margin-top: 0.5em;
+      margin-left: 0.5em;
+      &:hover {
+        cursor: pointer;
+        transform: scale(1.1);
+      }
+    }
+    .folder-sub-menu {
+      margin-left: auto;
+    }
+    .or {
+      margin-right: 0.5em;
+    }
+    .date-navigator {
+      margin-left: auto;
+      margin-right: auto;
+      input {
+        margin-top: 0.25em;
+      }
+    }
+  }
+  .table {
+    width: 100%;
+    .group {
+      background-color: #f9f9f9 !important;
```

Comment:
One day it would actually be nice to use themes correctly.  I was hoping someone would clean up the global css colours and such so that we could consistently use them in our styled components.  You don't need to do this here, I'm just highlighting it.